src/SimpleBayesianConsensusCaller.cpp

```diff
@@ -288,10 +292,10 @@ void SimpleBayesianConsensusCaller::loadConfiguration(ifstream& matrixFile){
             if (tokens[0] == "Name"){
```

Comment:
Do we now here that `tokens` is not empty?

Reply:
I believe the tokenizer will always return something if given a non-empty string, which is guaranteed in this case. If it contains no separators, it will just return the string as one token.